scripts/clean_reprocess_verify.py

```
@@ -0,0 +1,349 @@
+    parser.add_argument("--single", type=str, default=None,
+                       help="Process a single source filename (partial match)")
+    parser.add_argument("--resume", action="store_true",
+                       help="Skip books that already have output AND results in batch_compare_results.json")
+    parser.add_argument("--only-verify", action="store_true",
+                       help="Re-run verification on existing output without re-running pipeline")
+    args = parser.parse_args()
+    
+    # Lock file to prevent duplicate runs
+    if LOCK_FILE.exists():
+        print(f"Lock file {LOCK_FILE} exists — another batch may be running.")
+        print(f"Delete it manually if you're sure no other process is active.")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)